packages/codegen-ui/lib/generate-form-definition/form-to-component.ts

```diff
@@ -13,66 +13,86 @@
   See the License for the specific language governing permissions and
   limitations under the License.
  */
-import { SchemaModel, ModelFields, isGraphQLScalarType } from '@aws-amplify/datastore';
 import {
   StudioComponent,
-  StudioForm,
   StudioComponentChild,
-  StudioComponentProperty,
-  DataStoreCreateItemAction,
-  DataStoreUpdateItemAction,
-  FixedStudioComponentProperty,
+  FormDefinition,
+  FormDefinitionElement,
+  FormStyleConfig,
+  StudioComponentProperties,
+  StudioFormStyle,
 } from '../types';
 
-// map the datastore schema fields into form fields
-export const mapFieldsToForm = (fields: ModelFields) => {
-  const formFields: StudioComponentChild[] = [];
-  Object.entries(fields).forEach(([fieldName, fieldValue]) => {
-    // TODO: expand studio component child to also support other non text fields
-    if (isGraphQLScalarType(fieldValue.type) && !fieldValue.isArray) {
-      formFields.push({
-        name: `${fieldName}Field`,
-        componentType: 'TextField',
-        properties: {
-          name: {
-            value: fieldName,
-          },
-          label: {
-            value: fieldName,
-          },
-          placeholder: {
-            value: `${fieldValue.type}`,
-          },
-          ...(fieldValue.isRequired && {
-            required: {
-              value: 'true',
-              type: 'boolean',
-            },
-          }),
-        },
-      });
-    }
-  });
+const DefaultStudioFormStyles = {
```

Comment:
I think we need to export this so UI can use it. @hein-j can you confirm this is it?

Reply:
Defaults can be found in the file generate-form-definition/helpers/defaults. I have a task out to myself to either export from there for Studio or have it be consumed from the form definition. Most likely the latter.

For codegen, Josue can just use the form definition output. It will have the defaults applied if StudioForm doesn't have the style values.

Reply:
see also this comment - https://github.com/aws-amplify/amplify-codegen-ui/pull/505#discussion_r918273278

Reply:
updatingj in the next commit to remove this so we can solely depend on the style coming from the form def.